pkg/users/index.html

```diff
@@ -81,28 +81,28 @@ <h1 translatable="yes">Account not available or cannot be edited.</h1>
         <span id="account-title"></span>
       </div>
       <div class="panel-body">
-        <table class="info-table-ct col-sm-3">
+        <table class="info-table-ct">
           <tr>
-            <td translatable="yes">Full Name</td>
+            <th translatable="yes">Full Name</td>
```

Comment:
```suggestion
            <th translatable="yes"><label for="account-real-name">Full Name</label></td>
```

Reply:
BTW: I was just looking into `<label>`s again this morning and found that it only applies to form controls... and only a subset of the elements. Basically, form elements that are not self-labelling.

[See **WCAG 2.0 H44**](https://www.w3.org/TR/WCAG20-TECHS/H44.html#H44-description), where note 1 has a list of types of elements that get label (input of type text, checkbox, radio, file, & password; textarea; and select). Note 2 has some example form elements that don't get labels (inputs of type submit or reset, image buttons, hidden inputs, and script buttons). But also non-form elements should not get `<label>`s too.

There's also [the labelable element chart in the HTML5 spec](https://www.w3.org/TR/html5/sec-forms.html#labelable-element), which details which elements get `<label>`.

As this is for an input, it's good to use `<label for="">` here. However, there are a few items further down that aren't form controls. In those cases, I'd suggest using `aria-labelledby` in the element that is being described and have it point to the ID of the labelling element (the `<th>` in this case). However, as it's a table with <th scope="row">, it might be enough.

[See **WCAG 2.0 H63** for more on using the scope attribute to associate header cells and data cells](https://www.w3.org/TR/WCAG20-TECHS/H63.html).

Reply:
Yeah, this stuff is annoying and hard to get right, so the more we can move to a framework where we can re-use widgets (in our case, React), the easier it gets for everyone. :wink:

Reply:
Moved translatable to the label too.